transformation/src/main/scala/org/broadinstitute/monster/hca/HcaUtils.scala

```diff
@@ -0,0 +1,119 @@
+package org.broadinstitute.monster.hca
+
+import com.spotify.scio.ScioContext
+import com.spotify.scio.coders.Coder
+import com.spotify.scio.io.ClosedTap
+import com.spotify.scio.values.SCollection
+import org.apache.beam.sdk.io.FileIO.ReadableFile
+import org.apache.beam.sdk.io.{FileIO, ReadableFileCoder}
+import org.apache.beam.sdk.io.fs.EmptyMatchTreatment
+import org.broadinstitute.monster.common.StorageIO
+import org.broadinstitute.monster.common.msg.{JsonParser, UpackMsgCoder}
+import upack.{Msg, Obj, Str}
+
+import scala.collection.mutable
+import scala.util.matching.Regex
+
+object HcaUtils {
```

Comment:
I don't see much point in having this separate object. The methods are fine to have in the pipeline-builder.